File: 2942.find-words-containing-character.cs
/*
 * @lc app=leetcode id=2942 lang=csharp
 *
 * [2942] Find Words Containing Character
 */

// @lc code=start
using System.Collections.Generic;

public class Solution {
    public IList<int> FindWordsContaining(string[] words, char x) {
        List<int> result = new List<int>();
        for (int i = 0; i < words.Length; i++) {
            if (words[i].Contains(x)) {
                result.Add(i);
            }
        }
        return result;
    }
}

// @lc code=end

File: 2942.find-words-containing-character.py
#
# @lc app=leetcode id=2942 lang=python3
#
# [2942] Find Words Containing Character
#

# @lc code=start
from typing import List


class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        n = len(words)
        result = []
        for i in range(n):
            if x in words[i]: result.append(i)
        return result
# @lc code=end

